docs/development.rst

```diff
@@ -10,9 +10,6 @@ the tools used to develop Girder.
 Configuring Your Development Environment
 ----------------------------------------
 
-In order to develop Girder, you should first refer to the :doc:`installation quickstart <installation-quickstart>` and
```

Comment:
This probably leaves the development docs with inadequate instructions for prereqs.

Reply:
Agreed. I think we can maybe redo this whole section with the aim of being able to point, say, a new intern at the page and have them up and running with our "preferred" development setup.

Reply:
Can you add a follow up issue to track this?

Reply:
Done: #3281